.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,53 @@
+name: "unit-tests"
+
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily at 00:15 UTC (the 15 is to avoid periods of high load)
+    - cron: "15 0 * * *"
+  workflow_dispatch:
+
+permissions: {}
+
+concurrency:
+  group: "${{github.workflow}}-${{github.ref}}"
+
+  # Cancel in-progress jobs for efficiency
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+    runs-on: "${{matrix.os}}"
+    steps:
+      - uses: "actions/checkout@v4"
+        with:
+          submodules: "recursive"
+
+      - uses: "actions/setup-python@v5"
+        with:
+          python-version: "3.10"
+
+      - name: "Install task"
+        run: "npm install -g @go-task/cli"
+
+      - if: "matrix.os == 'macos-latest'"
+        name: "Install coreutils (for md5sum)"
+        run: "brew install coreutils"
+
+      - name: "Log tool versions"
+        run: |-
+          md5sum --version
+          python --version
+          tar --version
+          task --version
+
+      - name: "Install project dependencies "
+        timeout-minutes: 10
+        continue-on-error: false
```

Comment:
Isn't this the default?

Reply:
I know `continue-on-error` is `false` by default, but I can find no sources saying the step `timeout-minutes` has any default values.